src/javascript/lib/landing_pages/ico.js

```diff
@@ -465,6 +466,19 @@ function setupCrowdin() {
     }
 }
 
+function checkUserSession() {
+    var signup_form        = document.getElementById('sign-up-section');
+    var account_exists_msg = document.getElementById('account_exists_message');
+    if (!localStorage.getItem('active_loginid')) {
```

Comment:
in client.js we have this function:
```
 const isLoggedIn = () => (
        !isEmptyObject(getAllAccountsObject()) &&
        get('loginid') &&
        get('token')
    );
```
can you add a similar check so we are really sure that client is logged in and keep it consistent?